solidity/random-beacon/contracts/RandomBeacon.sol

```diff
@@ -146,6 +146,11 @@ contract RandomBeacon is Ownable {
     ///         operator affected.
     uint256 public relayEntryTimeoutNotificationRewardMultiplier;
 
+    /// @notice This amount is required to protect against griefing the staking
+    ///         contract and individual applications are allowed to require higher
+    ///         minimum authorization if necessary.
```

Comment:
Hmm... I could have missed something but how does it protect against griefing the staking contract? 🤔 

Minimum stake in T staking contract does protect against it because it ensures no dust stakes can be used to block operator addresses: imagine someone trying to stake and some malicious party frontrunning them with 1e-18 T. But I don't see how this value protects against it. It's just a minimum authorization we require and it needs to be high enough so that slashing for malicious DKG result or relay entry timeout can be executed - worth mentioning it in docs.

Reply:
changed in 36d98c5

I looked at T Staking contract and based my prev description on it. But yes, I see now that it's different.